.github/workflows/deploy-avax-strategy.yaml

```
@@ -0,0 +1,85 @@
+      name: avalanche
+      url: https://term-finance.github.io/yearn-v3-term-vault/
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - uses: foundry-rs/foundry-toolchain@v1
+      - run: forge install
+      - run: forge build
+      - run: forge tree
+      - run: forge script script/Strategy.s.sol:DeployStrategy --rpc-url $RPC_URL --broadcast --gas-price  500000000000  --verify --verbosity 4
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:61: Double quote to prevent globbing and word splitting

(shellcheck)